frontend/pages/login.vue

```diff
@@ -93,10 +93,13 @@ export default {
       });
     },
     async userLogin() {
+      const { username, password } = this.login;
       try {
         await this.$store.dispatch("entities/deleteAll");
         await this.$auth.loginWith("authProvider", {
-          data: `username=${this.login.username}&password=${this.login.password}`,
+          data: `username=${encodeURIComponent(
+            username
+          )}&password=${encodeURIComponent(password)}`,
```

Comment:
code cleanning : it's better to not do too much things in one line => create string from factory function